adagucserverEC/CDrawImage.h

```diff
@@ -101,6 +101,37 @@ class CColor {
   }
 };
 
+class RectangleText {
```

Comment:
Can you move this one to its own file?

Reply:
Also had to move CColor to it's own file for this.